editoast/src/views/infra/auto_fixes.rs

```diff
@@ -765,4 +793,45 @@ mod test {
             obj_type: ObjectType::BufferStop,
         })));
     }
+
+    #[rstest::rstest(endpoint, bs_position,
+        case(&Endpoint::Begin, 0.),
+        case(&Endpoint::End, 100.),
+    )]
+    fn missing_track_extremity_buffer_stop_fix(endpoint: &Endpoint, bs_position: f64) {
+        let mut cache = InfraCache::default();
+        let track = TrackSectionCache {
+            obj_id: "track_id".into(),
+            length: 100.,
+            bbox_geo: BoundingBox {
+                0: (0., 0.),
+                1: (0., 0.),
+            },
+            bbox_sch: BoundingBox {
+                0: (0., 0.),
+                1: (0., 0.),
+            },
+            ..Default::default()
+        };
+        cache.add(track.clone()).unwrap();
+        let error = InfraError::new_missing_buffer_stop(&track, *endpoint);
+
+        let operations = get_operations_fixing_error(&error, &cache).unwrap();
+        if let Operation::Create(boxed_buffer_stop) = operations.first().unwrap() {
```

Comment:
Could you expect() here instead?

Reply:
Probably yeah. This code disappeared while reformating the unit test, but thanks for that ! :)